test/unit/spec/v2/models/AppState_spec.js

```diff
@@ -5,6 +5,7 @@ import XHRAuthenticatorChallengOktaVerify
 import XHRAuthenticatorEnrollOktaVerify
   from '../../../../../playground/mocks/data/idp/idx/authenticator-enroll-ov-sms.json';
 import { FORMS_FOR_VERIFICATION, FORMS_WITHOUT_SIGNOUT } from 'v2/ion/RemediationConstants';
+import { jest } from '@jest/globals';
```

Comment:
This is a new pattern, did you editor do this?

Reply:
yeah that could have been the problem.. Reverted that and reran on bacon